src/libpriv/rpmostree-core.cxx

```diff
@@ -2503,10 +2503,15 @@ ht_insert_path_for_nevra (GHashTable *ht, const char *nevra, char *path, gpointe
 }
 
 /* This is a lighter version of calculations that librpm calls "file disposition".
- * Essentially, we determine which file removals/installations should be skipped. The librpm
- * functions and APIs for these are unfortunately private since they're just run as part of
- * rpmtsRun(). XXX: see if we can make the rpmfs APIs public, but we'd still need to support
- * RHEL/CentOS anyway. */
+ * Essentially, we determine which file removals/installations should be skipped. For
+ * example:
+ * - if removing a pkg owning a file shared with another pkg, the file should not be deleted
+ * - if adding a pkg owning a file that already exists in the base (or another added pkg),
+ *   and they are both "coloured", we need to pick the preferred one
+ *
+ * The librpm functions and APIs for these are unfortunately private since they're just run
+ * as part of rpmtsRun(). XXX: see if we can make the rpmfs APIs public, but we'd still need
+ * to support RHEL/CentOS anyway. */
```

Comment:
It doesn't matter here but I'd s/RHEL/RHEL8/ at least mentally - remember "RHEL" is older until suddenly it isn't :smile:

Reply:
It's an old comment, but yeah misleading now given CentOS Stream. Had to respin for CI so dropped that bit entirely!